pkg/apis/numaflow/v1alpha1/vertex_template.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+type VertexTemplate struct {
+	// +optional
+	AbstractPodTemplate `json:",inline" protobuf:"bytes,1,opt,name=abstractPodTemplate"`
```

Comment:
initially did inline to be consistent with `daemon`/`job`, but just looked back at issue and seems like maybe `json:"podTemplate,omitempty"` is desired to be consistent with `containerTemplate`/`initContainerTemplate`.  let me know if i should make that change here & the docs.

Reply:
just to confirm, i should leave `daemon` and `job` unchanged (keep those inline)?